common/app/common/JsMinifier.scala

```diff
@@ -65,8 +60,8 @@ object JsMinifier {
     if (result.warnings.isEmpty && result.errors.isEmpty && result.success) {
       compiler.toSource
     } else {
-      val errors: List[String] = result.errors.map(_.toString).toList
-      val warnings: List[String] = result.warnings.map(_.toString).toList
+      val errors: List[String] = result.errors.toArray().map(_.toString).toList
+      val warnings: List[String] = result.warnings.toArray().map(_.toString).toList
```

Comment:
`errors` and `warnings`  have changed from `JSError[]` to `ImmutableList<JsErrors>` in
https://github.com/google/closure-compiler/commit/9b1995559e58a95bc96cdbb39716b92887f564d7